### ISSUE TITLE
Dark mode and prefers-color-scheme

### ISSUE THREAD

**Issue by Kernald** [JugglerX/hugo-whisper-theme#33]
It would be nice having a dark mode. Using [`prefers-color-scheme`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme), there would be no need on a button to toggle or anything like that, making this just dependent on providing dark variants for colours.

**Comment by blacktop** [comment]
+1000 !!!

**Comment by JugglerX** [comment]
Yes it would be nice, but unfortunately I do not plan to add this feature.

It will need to be added in a PR from the community.

**Comment by zhan-ge** [comment]
+1000 !!!

**Comment by AsenHu** [comment]
I will probably implement this feature, but I won't be making a custom color profile.
